src/app/[locale]/seller/sale-items/[id]/page.tsx

```diff
@@ -170,18 +170,6 @@ export default function BuyFromSellerForm({ params }: { params: { id: string } }
             </span>
             <span>{sellerInfo ? sellerInfo.user_name : ''}</span>
           </div>
-          <div className="text-sm mb-3">
-            <span className="font-bold">
-              {t('SHARED.USER_INFORMATION.PHONE_NUMBER_LABEL') + ': '}
-            </span>
-            <span>{sellerSettings ? sellerSettings.phone_number : ""}</span>
```

Comment:
<span>{sellerSettings.phone_number ? sellerSettings.phone_number : ""}</span>
since default value is now null (if you still want to display it)